### PR TITLE
Add bors icon

### PR DESCRIPTION
**Pull request by notriddle**
https://discourse.janitor.technology/t/id-like-to-add-bors-ng-to-the-janitor/205

### CODE REVIEW

**Comment by jankeromnes** [comment]
Exciting! Are you planning to add bors to janitor, or implement some kind of integration? :smile: 

Also, @nt1m please have a look when you have some time.

**Comment by notriddle** [comment]
I'm planning to add bors to the Janitor as a perfectly ordinary project.

(I've got nothing against some kind of integration, but I don't have anything in mind right now to integrate).

File: static/img/bors.svg [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+   width="12.293mm"
+   height="13.123mm"
+   version="1.1"
+   viewBox="0 0 12.293362 13.1233"
+   id="svg10"
+   sodipodi:docname="i.svg"
+   inkscape:version="0.92pre4 15278">
+  <defs
+     id="defs14" />
+  <sodipodi:namedview
```
[the remark this answers is not in the document]
Comment (notriddle) [reply static/img/bors.svg]: Done.

File: static/img/bors.svg [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="12.293mm"
+   height="13.123mm"
```
[the remark this answers is not in the document]
Comment (notriddle) [reply static/img/bors.svg]: Done.

**Comment by nt1m** [comment]
@notriddle Please feel free to "Squash and merge" once the status checks are finished :)

**Pull request merged**